airlock/business_logic.py

```diff
@@ -877,19 +879,20 @@ def all_files_approved(self):
             for request_file in filegroup.output_files
         )
 
-    def all_files_reviewed(self):
+    def all_files_reviewed_by_reviewer(self, reviewer: User) -> bool:
         return all(
-            request_file.reviewed()
-            for filegroup in self.filegroups.values()
-            for request_file in filegroup.output_files
+            rfile.get_status_for_user(reviewer) is not None
```

Comment:
as noted by @rebkwok, if the status is UNDECIDED, this will probably do the wrong thing